projects/admin/src/app/record/brief-view/documents-brief-view/documents-brief-view.component.html

```diff
@@ -29,7 +29,11 @@ <h5 class="mb-0 card-title">
     </h5>
     <ng-container class="card-text">
       <!-- contribution -->
-      <shared-contribution [contributions]="record.metadata?.contribution" [limitRecord]="3"></shared-contribution>
+      <shared-contribution
+        [contributions]="record.metadata?.contribution"
+        [limitRecord]="3"
+        [import]="type.startsWith('import_')"
```

Comment:
I'm not fan about this param name... it reflects a specific case than `shared_contribution` doesn't need to know.
`shared-contribution` needs to know if the contribution link should be activated or not ; so I would named the parameter as `[activate_link]` or something like that (with a default value set to True)